src/adhocracy_spd/adhocracy_spd/workflows/test_digital_leben.py

```diff
@@ -1,44 +1,17 @@
+"""
+Test the digital leben workflow, which is now in core named
+'debate'. Acts as a regression test.
+
+"""
 from pyramid import testing
 from pytest import fixture
 from pytest import mark
 from webtest import TestResponse
 
+from adhocracy_core.utils.testing import add_resources
```

Comment:
How do you change the workflow for one process? The process is created via the JSON file and the workflow name key from the sheet is readonly.